randovania/generator/pickup_pool/pickup_creator.py

```diff
@@ -137,17 +137,22 @@ def create_nothing_pickup(resource_database: ResourceDatabase, model_name: str =
     )
 
 
-def create_visual_etm() -> PickupEntry:
+def create_visual_nothing(game: RandovaniaGame = RandovaniaGame.METROID_PRIME_ECHOES,
```

Comment:
I'd make the two first arguments not have defaults

Reply:
What values do the other games expect? AFAIK every game is still built around the idea of having an ETM passed in?
Wondering whether i should keep it that way for now and put it off to the future, or just fix it while i'm already here.

Reply:
These could likely be fields in the PickupDatabase. The PatchDataFactory has a `pickup_db` field even.

If refactoring, it'd be nice to rename all these `visual_etm` arguments all over the place too, but I abstained in requesting it :P